packages/manager/modules/error-page/src/error.html

```diff
@@ -0,0 +1,53 @@
+<div class="h-100 ovh-manager-error-page">
+    <div class="container-fluid h-100 position-absolute overflow-auto">
```

Comment:
You do need to apply `h-100` on this element and its parent ?

Reply:
I think we reached a point that to display a few elements we shouldn't too many nested elements (5 nested elements until the image is displayed). Do you agree?

Maybe related to the parent layout but I think we should handle it in a different way to reduce as much as we can the nested elements.

```txt
IMG (centered)

Title (centered)

Alert message (centered)

Three actions button|anchor on a reversed row
```

Reply:
This due to bootstrap syntax and responsive handling. I'll check if some of the `div` elements could be removed.